packages/renderer/src/lib/table/Table.svelte

```diff
@@ -143,9 +143,9 @@ function setGridColumns() {
       class="grid grid-table gap-x-0.5 mx-5 h-12 bg-charcoal-800 hover:bg-zinc-700 rounded-lg mb-2"
       animate:flip="{{ duration: 300 }}"
       role="row">
```

Comment:
Adding a `aria-label="{object.name}"` into a row element would be also super helpful.